src/e2e-test/features/bigquery/source/BigQueryToGCS_WithMacro.feature

```diff
@@ -0,0 +1,40 @@
+@BigQuery_Source
+Feature: BigQuery source - Verification of BigQuery to GCS successful data transfer with macro arguments
+
+  @MACRO_TEST @CMEK @BQ_SOURCE_TEST @GCS_SINK_TEST
+  Scenario:Validate successful records transfer from BigQuery to GCS with macro arguments
+    Given Open Datafusion Project to configure pipeline
+    When Source is BigQuery
+    When Sink is GCS
+    Then Open BigQuery source properties
+    Then Enter BigQuery property reference name
+    Then Enter BigQuery property projectId as macro argument "bqProjectId" value "projectId"
```

Comment:
Separate out the steps of passing key and value.
What I mean is, we should first setup all configs with macro. 
Then after deploying the pipeline or before preview run set the runtime arguments.

Reply:
This comment applies to all the steps.

Reply:
Steps separated.